.cargo/config.toml

```diff
@@ -34,7 +34,13 @@ rustflags = [
 rustflags = [
     "--cfg",
     "tokio_unstable",
-    # uncomment the following two lines to enable `TaskLocalAlloc`
+
+    # Register the `rw` tool for referencing the custom lints in attributes like `#[allow(rw::xx)]`.
+    # This is essentially the same as `#![..]` in each crate root.
```

Comment:
Oh, can we use this to enable features? 🫣

Reply:
Yeah, really interesting! But it seems painful to also override `RUSTDOCFLAGS`. 😕

Reply:
Seems not to work for doc tests. I'll manually register it. 🤡